### ISSUE TITLE
Google voice acting stopped working

### ISSUE THREAD

**Issue by disasdter** [seanhweb/Twitch-Text-to-Speech#40]
Google voiceover doesn't work, only Microsoft, what could be the problem? Tried VPN, only Google doesn't work either
